Vhar 8207 piilota akilliset toteumista

rckopteri opened the pull request.

rckopteri committed:
Poista kasin_lisattava_maara akillisilta ja lisätöiltä

rckopteri committed:
Poista äkilliset ja lisätyöt toteumat/tehtävät sivulta

Sekä toteuma-lomakkeelta, jolta voi lisätä käsin toteumia tehtävälle.

Palauta kuitenkin Kolmansien osapuolien aiheuttamien vahinkojen tehtävät lomakkeelle.

rckopteri committed:
Paranna funktioiden, tietokantahakujen ja endpointtien nimeämistä

Vieläkään nimet eivät ole täydellisiä, mutta kuvaavat ehkä enemmän, että mitä yritetään tehdä

rckopteri committed:
Poista lisätöiden testit käsin lisättävän toteuman testeistä

rckopteri committed:
Päivitä muuttunut endpoint cypress testeihin

rckopteri committed:
Lisää LEFT JOIN kuluille, koska kaikilla kulu_kohdistus riveillä ei ole tehtäväryhmää

jpluomajoki-solita commented on test/clj/harja/palvelin/palvelut/toteumat_lisatyot_akilliset_test.clj:
```diff
@@ -64,7 +64,7 @@
                                         :toteuma-tehtava-id nil
                                         }]})
 
-(def default-akillinen-hoitotyo {:urakka-id (hae-oulun-maanteiden-hoitourakan-2019-2024-id)
```
Olisiko parempi poistaa kokonaan?

rckopteri replied:
Poistin kokonaan.

jpluomajoki-solita reviewed (approved):
lgtm, pitäisikö vanhentuneet testit poistaa kommentoinnin sijaan?

solita-staskila reviewed (approved):
> lgtm, pitäisikö vanhentuneet testit poistaa kommentoinnin sijaan?

Samaa mieltä, onko jokin pointti säilyttää testien koodi, jos lisätöille ei voi enää lisätä tai muokata toteumia?

rckopteri committed:
Palautetaan lisätyöt, kun eihän niitä saanutkaan poistaa.

Ne vaan piti jättää 9 Muut otsikon alle, mutta ei 5 Liikenteen varmistamisen alle.
